### DISCUSSION TITLE
16 Plots are invalid, you can now delete them

### DISCUSSION THREAD

**Discussion by oytp** [Chia-Network/chia-blockchain#3640] [Support]
Hi there.

I've been plotting since 28th April, and today I got 16 invalid plots, they start from the first one till yesterday one, 16 at all. I used 1.1.2 Client for plotting.
How can I fix them? Or what should I do now?

![image](https://user-images.githubusercontent.com/83637165/116976567-04460100-acca-11eb-83cc-7bcf2a3839e8.png)

**Comment by oytp** [comment]
I closed the Client, then rebooted PC. After that all the plots are valid now.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
